### PR TITLE
Rename tool oegenerate to oecert

### LINKED ISSUE #4027: oegenerate doesn't generate open enclaves
The name "oegenerate" is misleading since it does not in fact generate open enclaves.
To have a good developer experience, rather than an confusing one, the naming is important.
If the tool does things specific to attestation, then it needs a name specific to attestation, not one that implies it generates projects.    oeedger8r generates stubs from IDL, and the VS and VS Code extensions generate projects from templates, and neither of those are called oegenerate, hence the confusion.

### PR DESCRIPTION
**Pull request by yentsanglee**
Per discussion in [SIG Attestation](https://hackmd.io/Xj6GpDSKSwuz5cZgQ0yg1A#19-May-2021-1000-AM-PDT), revert tool "oegenerate" name to "oecert".

Fix #4027 

Signed-off-by: Yen Lee <yenlee@microsoft.com>

### CODE REVIEW

**Comment by yentsanglee** [comment]
bors try

**Review by mingweishih — APPROVED** [review]

**Review by ryanhsu19 — APPROVED** [review]
Looks good. Thanks for the work!

File: tools/oecert/host/host.cpp [hunk 1]
```diff
@@ -553,11 +552,11 @@ static oe_result_t _process_params(oe_enclave_t* enclave)
     oe_result_t result = OE_FAILURE;
 
 #if defined(__linux__)
-    char* sgx_aesm_env = getenv(SGX_AESM_ADDR);
+    char* sgx_asem_env = getenv(SGX_AESM_ADDR);
```
Comment (dthaler) [inline tools/oecert/host/host.cpp:555]: This typo correction need not be reverted, but if it's easiest to revert collectively, it's ok.

**Review by dthaler — APPROVED** [review]

**Review by shnwc — APPROVED** [review]

**Comment by yentsanglee** [comment]
This needs to be discussed in SIG Attestation meeting. 'oegenerate' has been released in 0.16.0. Reverting back to "oecert" does not mean much anymore. Will keep this PR open until a new conclusion is formed. 

Please do not merge this PR.

**Comment by yentsanglee** [comment]
Closing this PR. Issue is now tracked by #4012

**Pull request closed without merging**